app/queries/intent_metrics_query.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+# Query to return the metrics on the intent selections of the students in a
+# specific draw.
+class IntentMetricsQuery
+  class << self
+    delegate :call, to: :new
+  end
```

Comment:
this should at least be documented because it was the first time I've actually seen this used, so it's likely to confuse others down the road